plugins/command_cedict.py

```diff
@@ -0,0 +1,120 @@
+import re
+
+from pyrogram import Client, filters
+from pyrogram.types import Message, \
+    InlineKeyboardMarkup, \
+    InlineKeyboardButton, \
+    CallbackQuery
+
+from tools.cedict import get_cedict_result
+from tools.utils import command_args_to_str
+
+CEDICT_ENTRIES_PER_PAGE = 10
+
+
+@Client.on_message(filters.command(["cedict", "dict", "dic"])
+                   & filters.incoming
+                   & ~filters.edited)
+async def command_cedict(_, message: Message):
+    """/cedict question - get translation from cedict"""
+    query = command_args_to_str(message.command)
+    if query:
+        first_page = 0
+        cedict_response = get_cedict_result(query,
+                                            first_page,
+                                            CEDICT_ENTRIES_PER_PAGE)
+
+        reply_markup = None
+        if cedict_response.has_next_page:
+            reply_markup = _make_next_page_button()
+
+        await message.reply(cedict_response.formatted_result,
+                            reply_markup=reply_markup, quote=True)
+    else:
+        await message.reply("Usage: \n<code>/cedict your query</code>"
+                            "\n\nFor example:\n<code>/cedict China</code>"
+                            "\n\nYou can also use <code>/dic</code> instead of"
+                            " <code>/cedict</code>",
+                            quote=True)
+
+
+@Client.on_callback_query(filters.regex("cedict_next_button_pressed")
+                          | filters.regex("cedict_previous_button_pressed"))
+async def callback_query_cedict_button_pressed(_,
+                                               callback_query: CallbackQuery):
+    current_page_first_entry_number = re.search(
```

Comment:
```
@Client.on_callback_query(
    filters.regex(r"^cedict_(previous|next)_button_pressed$")
)
async def query_cedict_button_pressed(_, cq: CallbackQuery):

```

yous should change the regex for proper match (with your current code, it will also match `foobarbaz_cedict_next_button_pressed_alphabeta`, because it does not include the start `^` and the end `$`)

you can just name the button callback data to `cedict_next` or `cedict_next_page`, and you can use `cq` the parameter name since it's obvious that it means callback query (type hint, and on_callback_query). but it's just naming, does not really matter.